app/scripts/modules/core/ci/jenkins/igor.service.js

```diff
@@ -8,10 +8,11 @@ module.exports = angular.module('spinnaker.core.ci.jenkins.igor.service', [
   require('../../config/settings.js'),
   API_SERVICE,
 ])
-  .factory('igorService', function (settings, API) {
+  .factory('igorJenkinsService', function (settings, API) {
 
     function listMasters() {
-      return API.one('v2').one('builds').get();
+      return API.one('v2').one('builds').get()
```

Comment:
Why not just keep one igorService and have a filter `listMasters('jenkins')` and `listMasters('travis')`